scripts/lint-ocaml

```diff
@@ -18,7 +18,7 @@ set -eu
 # Note however that the pre-commit CI action may loop forever if
 # you leave those commands uncommented (no idea why, go figure again).
 
-if [[ -v SKIP_OCAMLFORMAT ]]; then
+if [[ -z ${SKIP_OCAMLFORMAT+set} ]]; then
```

Comment:
:)

Just smiling. You can resolve this.

Reply:
Ah I didn't even mean to commit this. I just keep having trouble getting bash 5.x to be reliably used instead of the default. 😔